Lack of documentation about Rust for compiling the extension

As freeBSD does not support any docker system apart from setting up VM and installing linux and installing docker in the linux I was compiling timescale-prometheus manually my last version is the 0.1.0-alpha.4 and the timescale_prometheus_extra was successfully compiled using gmake, today I update the version to 0.1.0-beta.4 and successfully compiled the timescale-prometheus but when I try to compiled and install the latest timescale_prometheus_extra I get the error as below: - 

``` 
root@dnsvgitnpm:~/install/timescale-prometheus/extension # gmake clean
Makefile:57: warning: overriding recipe for target 'clean'
/usr/local/lib/postgresql/pgxs/src/makefiles/pgxs.mk:355: warning: ignoring old recipe for target 'clean'
rm -f timescale_prometheus_extra.so   libtimescale_prometheus_extra.a  libtimescale_prometheus_extra.pc
rm -f src/support.o src/support.bc
rm -f timescale_prometheus_extra.so
cargo clean
gmake: cargo: No such file or directory
gmake: *** [Makefile:59: clean] Error 127
root@dnsvgitnpm:~/install/timescale-prometheus/extension # gmake
Makefile:57: warning: overriding recipe for target 'clean'
/usr/local/lib/postgresql/pgxs/src/makefiles/pgxs.mk:355: warning: ignoring old recipe for target 'clean'
cc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -Wno-unused-command-line-argument -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing  -fPIC -DPIC -DINCLUDE_PACKAGE_SUPPORT=0 -MMD -I. -I./ -I/usr/local/include/postgresql/server -I/usr/local/include/postgresql/internal  -I/usr/local/include -I/usr/local/include -I/usr/local/include  -c -o src/support.o src/support.c
cargo build --release 
gmake: cargo: No such file or directory
gmake: *** [Makefile:54: target/release/libtimescale_prometheus_extra_rs.a] Error 127
root@dnsvgitnpm:~/install/timescale-prometheus/extension # gmake install
Makefile:57: warning: overriding recipe for target 'clean'
/usr/local/lib/postgresql/pgxs/src/makefiles/pgxs.mk:355: warning: ignoring old recipe for target 'clean'
cargo build --release 
gmake: cargo: No such file or directory
gmake: *** [Makefile:54: target/release/libtimescale_prometheus_extra_rs.a] Error 127
```
any Idea what could be the problem. 
from my poor skill in compiling things I can see that cargo cannot find some file.. 

> gmake: cargo: No such file or directory

but does not mentioned what file

--- a/extension/Readme.md
+++ b/extension/Readme.md
@@ -7,7 +7,12 @@ cause it to perform better.
 
 ## Requirements ##
 
-This extension requires Postgres version 12 or newer.
+To run the extension:
+- PostgreSQL version 12 or newer. 
+
+To compile the extension:
+- Header files for PostgreSQL version 12 or newer. Headers are included in the `postgresql-server-dev-12` package in Debian or Ubuntu. For other platforms see the PostgreSQL [download page](https://www.postgresql.org/download/).
+- [Rust compiler](https://www.rust-lang.org/tools/install)
 
 ## Installation ##
 
@@ -17,3 +22,8 @@ The extension is installed by default on the
 To compile and install from source run: `make && make install`.
 
 This extension will be created via `CREATE EXTENSION` automatically by the timescale-prometheus connector and should not be created manually.
+
+## Common Compilation Issues ##
+
+- `cargo: No such file or directory` means the [Rust compiler](https://www.rust-lang.org/tools/install) is not installed
+- `postgres.h` not found means that you are missing the PostgreSQL header files. If using Debian or Ubuntu simply install the `postgresql-server-dev-12` package. For other platforms see the PostgreSQL [download page](https://www.postgresql.org/download/).

